client/js/jsonedit.js

```diff
@@ -198,6 +198,15 @@ const jeCommands = [
       jeSetAndSelect([]);
     }
   },
+  {
+    id: 'je_grid',
+    name: 'grid element',
+    context: '^.* ↦ grid',
+    call: async function() {
+      jeStateNow.grid.push('###SELECT ME###');
```

Comment:
We briefly talked about this in the last call and we thought it might be a good idea to add `x` and `y` here by default (maybe preset to the widget's dimensions? Maybe also `offsetX` and `offsetY` but those might be better left optional.

Reply:
What harm would there be in automatically including the offsets with default value of 0? That way you wouldn’t have to type them in. And then wouldn’t they just disappear when you clicked out of them came back to the widget because they were defaults?

Reply:
> And then wouldn’t they just disappear when you clicked out of them came back to the widget because they were defaults?

No, that only happens to top-level properties.

Reply:
Then don’t add them by default, but make a button available to add them like with other properties. Maybe that was already done in this PR. Haven’t been able to look yet.

Reply:
> Maybe that was already done in this PR.

It was.

Reply:
Added defaults for x and y, but did not add other grid attributes.